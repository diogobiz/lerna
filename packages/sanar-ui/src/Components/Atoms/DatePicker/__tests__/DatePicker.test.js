import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESDatePicker from '../'

it('renders correctly', () => {
    const component = <ESDatePicker />

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
