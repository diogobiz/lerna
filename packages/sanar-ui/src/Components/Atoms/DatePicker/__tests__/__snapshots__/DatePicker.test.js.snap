// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="es-date-picker ant-calendar-picker"
  onBlur={[Function]}
  onFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  style={Object {}}
>
  <div
    onClick={[Function]}
    onKeyDown={[Function]}
    onMouseDown={[Function]}
    onTouchStart={[Function]}
  >
    <input
      className="ant-calendar-picker-input ant-input"
      placeholder="Select date"
      readOnly={true}
      value=""
    />
    <i
      className="anticon es-eva-icon ant-calendar-picker-icon es-eva-icon--medium"
    >
      <i
        data-eva="calendar-outline"
        data-eva-fill="#fff"
        data-eva-height="18px"
        data-eva-width="18px"
      />
    </i>
  </div>
</span>
`;
